openwisp_users/models.py

```diff
@@ -25,6 +26,24 @@ class Organization(BaseOrganization, AbstractOrganization):
     class Meta(AbstractOrganization.Meta):
         swapper.swappable_setting('openwisp_users', 'Organization')
 
+    def add_user(self, user, **kwargs):
+        """
+        overriding this method of ``AbstractOrganization``
+        to remove organization owner creation logic as it is
+        being handled separately by a signal of this project.
+        """
+
+        users_count = self.users.all().count()
+        if users_count == 0:
+            is_admin = True
+        
+        OrganizationUser = swapper.load_model('openwisp_users', 'OrganizationUser')
+        org_user = OrganizationUser.objects.create(
+            user=user, organization=self, is_admin=is_admin
+        )
```

Comment:
let's just return here, eg:

```suggestion
        return OrganizationUser.objects.create(
            user=user, organization=self, is_admin=is_admin
        )
```